vcs.mk

```diff
@@ -0,0 +1,45 @@
+PERMISSIVE_ON=+permissive
```

Comment:
We can now delete this from here since your other PR moves this into `variables.mk`.

Reply:
Fixed.